setup.sh

```diff
@@ -359,9 +359,7 @@ function synchronize_files()
   if command_exists 'zsh'; then
     # Add tabcompletion to zshrc
     if [[ -f "${HOME}/.zshrc" || -L "${HOME}/.zshrc" ]]; then
-      safe_append '# Enable bash completion for zsh' "${HOME}/.zshrc"
-      safe_append 'autoload bashcompinit && bashcompinit' "${HOME}/.zshrc"
-      append_bashcompletion '.zshrc'
+      append_zshcompletion
```

Comment:
We need to remove the unnecessary legacy lines from the `.zshrc` file.

Reply:
Thanks a lot for the feedback! Strangely I could not reproduce the bug and my zsh version is also 5.9 (however my setup is fairly fresh and maybe that is why).

@kwy95 removing the unnecessary legacy lines from `.zshrc` did the trick?